packages/create-cloudflare/e2e/tests/cli/cli.test.ts

```diff
@@ -555,7 +555,7 @@ describe("Create Cloudflare CLI", () => {
 					    npm create cloudflare -- --framework next -- --ts
 					    pnpm create cloudflare --framework next -- --ts
```

Comment:
nit: next is probably not the best example here as it doesn't support `--ts`

Reply:
https://github.com/cloudflare/workers-sdk/pull/11577 🙂